### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by KAfable** [lumien231/Random-Things#312]
Was trying to put items into a thermal expansion cache when I crashed - 

https://hastebin.com/akulagedej.sql

**Comment by lumien231** [comment]
Can you reproduce this or did it only happen once?

**Comment by KAfable** [comment]
Not reliably, but this was the second time. Both times happened when I was near the Player Interface physically and breaking/placing AE2 Machines/cables/busses near it. The other crash also happened when I tried to wrench a Thermal Expansion Cache right on top of the Player Interface.

**Comment by lumien231** [comment]
I'm guessing you have an AE Export Bus or something connected to one?

**Comment by KAfable** [comment]
Yeah, I actually have an ME Import bus (filtered for dragon hearts) attached to the side that represents the 27 slot inventory.
